packages/expo-screen-orientation/ios/ScreenOrientationUtilities.swift

```diff
@@ -0,0 +1,111 @@
+import ExpoModulesCore
+
+internal func isPad() -> Bool {
+  return UIDevice.current.userInterfaceIdiom == .pad
+}
+
+// https://medium.com/@cafielo/how-to-detect-notch-screen-in-swift-56271827625d
+internal var doesDeviceHaveNotch = {
+  var bottomSafeAreaInsetIsPositive = false
+  EXUtilities.performSynchronously {
+    bottomSafeAreaInsetIsPositive = (UIApplication.shared.delegate?.window??.safeAreaInsets.bottom ?? 0.0) > 0.0
```

Comment:
```suggestion
    bottomSafeAreaInsetIsPositive = (UIApplication.shared.delegate?.window?.safeAreaInsets.bottom ?? 0.0) > 0.0
```
I didn't know that you can chain the `?` operator 😅

Reply:
The delegate returns a `UIWindow??` type when referencing `delegate.window` so the chaining of `?` operator is necessary. It's unclear to me why this is the case though.